starter-repos/app-template/config/base/app/deployment.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
```

Comment:
The golang-template example uses remote bases. @viglesiasce is it preferred to use remote bases here?

Reply:
I suggest to use the local base since that is recommended by kustomize. The remote base is mainly for demo and quick trial purpose.